mkdocs.yml

```diff
@@ -2,15 +2,16 @@ docs_dir: docs/book
 site_dir: docs/html
 pages:
     - index.md
-    - Introduction: intro.md
-    - Usage: usage.md
-    - "Authorization server": authorization-server.md
```

Comment:
One note: because these pages are _already published_, we need to provide them in the repository as _redirects_. You can see how this is done in the zend-expressive repo; it involves having pages with a JS redirect, and then having `_`-prefixed pages in the `mkdocs.yml` file that point to them (so they don't end up in the TOC).